spec/factories/packages.rb

```diff
@@ -36,7 +37,7 @@
     trait :received do
       state "received"
       received_at { Time.now }
-      inventory_number "123456"
+      inventory_number {rand(1000000).to_s.rjust(6,'0')}
```

Comment:
Can you turn this into a sequence? - see spec/factories/sequences.rb Then it can be re-used above.